userguide/aviate/aviate-health.adoc

```diff
@@ -284,8 +284,8 @@ This command generates a diagnostic file with the `killbill_configuration.data`
 In order to include the logs in the diagnostic file, the following parameters need to be specified:
 
 * withLogs=true
-* logsDir=path of the directory from which to include the log files
-* logsFilenames=name of the log file that needs to be included. A separate `logsFilenames` parameter needs to be specified corresponding to each log file.
+* logsDir=path of the directory from which to include the log files. If unspecified, this defaults to the value of `com.killbill.billing.plugin.aviate.logsPath` property. If this property is not specified, it defaults to  `TOMCAT_HOME/logs`  where `TOMCAT_HOME` is determined via the `catalina.base` system property
+* logsFilenames=name of the log file that needs to be included. A separate `logsFilenames` parameter needs to be specified corresponding to each log file. If unspecified, this defaults to `catalina.out`.
```

Comment:
Should we say that if we want multiple log files, we should include this param multiple times (as shown in example below).

Reply:
Addressed in 3c732df